instance/outputs.tf

```diff
@@ -34,10 +34,6 @@ output "instance_private_ip" {
   value = ["${concat(aws_instance.instance.*.private_ip, aws_instance.instance_no_ebs.*.private_ip)}"]
 }
 
-output "instance_vpc_security_group_ids" {
-  value = ["${concat(aws_instance.instance.*.instance_vpc_security_group_ids, aws_instance.instance_no_ebs.*.instance_vpc_security_group_ids)}"]
```

Comment:
Don't remove this section, but update the parameter name from `instance_vpc_security_group_ids` to `vpc_security_group_ids`.

Reply:
Why would you want that output?

Reply:
Can you give me a use-case for that output?